java/google/registry/rde/imports/XjcToContactResourceConverter.java

```diff
@@ -68,6 +75,18 @@ public StatusValue apply(XjcContactStatusType status) {
 
   /** Converts {@link XjcRdeContact} to {@link ContactResource}. */
   static ContactResource convertContact(XjcRdeContact contact) {
+    ofy().save().entity(
+        new HistoryEntry.Builder()
+            .setType(HistoryEntry.Type.RDE_IMPORT)
+            .setClientId(contact.getClID())
+            .setTrid(generateTridForImport())
```

Comment:
These don't have to be random or one use. You can use the same one everywhere or append the time to it.